### PR TITLE
#1 - buildable skeleton

### PR DESCRIPTION
**Pull request by filfreire**
/cc @jmromanos

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,11 @@
...
+[![License: MIT](https://img.shields.io/badge/License-MIT-yellow.svg)](https://opensource.org/licenses/MIT)
+
+# tranquility-cucumber-example
+
+an example of [tranquility](https://github.com/tranquility-bdd/tranquility) + cucumber for golang ([godog](https://github.com/DATA-DOG/godog))
+
+## how to build & run (temporary)
+
+1. clone the repo
+2. `$ go get github.com/DATA-DOG/godog`
```
Comment (jmromanos) [inline README.md:10]: Shouldn't we add also `go get github.com/tranquility-bdd/tranquility`?

File: example_test.go [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+func iSetParamAparamBAsAnd(arg1, arg2 string) error {
+	paramA = arg1
+	paramB = arg2
+	return nil
+}
+
+func iMakeAnHttpRequestToGetPostmanecho() error {
+	var url = "https://postman-echo.com/get?foo1=" + paramA + "&foo2=" + paramB
+	var action = tranquility.Action{"GET", url, nil, nil, ""}
```
Comment (jmromanos) [inline example_test.go]: I Think that this can be also done as ```tranquility.Action{Method: "GET", URL:url}``` that it is much easier to read.

**Review by jmromanos — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (filfreire) [reply README.md:10]: improved, thanks!

File: example_test.go (hunk 2 above)
Comment (filfreire) [reply example_test.go]: improved

**Pull request merged**